libclamav/libmspack.c

```diff
@@ -290,7 +290,7 @@ static void mspack_fmap_message(struct mspack_file *file, const char *fmt, ...)
         buff[strlen(buff)]     = '\n';
         buff[strlen(buff) + 1] = '\0';
 
-        fputs(buff, stderr);
+        cli_eprintf(buff);
```

Comment:
Any function that utilizes a format string should never be passed the content as that format string (lest it contain escapes like `%s` that would cause an unintended stack read).  I think you probably meant:
``` c
cli_eprintf("%s", buff);

Reply:
WHOOOPS!  Yes thanks for the catch.

Reply:
Yeah looks it should be this 
```c
        clrs_eprint(buff);
```
instead of 
```c
        cli_eprintf(buff);
```
since the buffer is already formatted.